src/styles/view-switcher.scss

```diff
@@ -0,0 +1,61 @@
+.view-switcher {
+  display: flex;
+  justify-content: center;
+
+  // These are the exact margins required to vertically align the top of the
+  // view switcher with the top of the Hide Highlights button to its left,
+  // and the top of the first annotation card below the view switcher with the
+  // top of the New Page Note button to its left.
+  margin-top: 1px;
+  margin-bottom: 6px;
+}
+
+.view-switcher__tab {
+  @include smallshadow
+
+  height: 30px; // Same height as used for Hide Highlights and New Page Note
+                // buttons to left of view switcher.
+  padding-left: 12px;
+  padding-right: 12px;
+
+  background: $white;
+  transition: background-color .1s linear;
+
+  border: 1px solid $gray-lighter;
+
+  touch-action: manipulation;
```

Comment:
> Makes tapping the button faster as browser doesn't need to delay generation of click event to wait and see if you're doing a double-tap.

That's not relevant in most web apps nowadays because browsers [disable the double tap gesture](https://webkit.org/blog/5610/more-responsive-tapping-on-ios/) by default if the page is mobile-ready. In the case of our sidebar the story could be more complex.

I would suggest we remove this until we are sure it does something we want and we've actually tested it.